Controllers/ToppingsController.js

```diff
@@ -5,9 +5,43 @@ const ObjectId = require("mongodb").ObjectId;
 
 const getAllToppings = (req, res) => {
   DbService.connectToDb(async (db) => {
-    const toppings = await ToppingsService.getAllToppings(db);
-    res.json(toppings);
+    const result = await ToppingsService.getAllToppings(db);
+    let jsonRes = ResponsesService.successful();
+    jsonRes.data = result;
```

Comment:
What if there are no toppings for whatever? I'd say that in that case it would still be a successful response type, because nothing failed. I'd suggest checking the result, and if it's empty changing the message in the response to reflect that.

That way it makes it clear for anyone using the API that it's not broken if they get back an empty array